src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -117,12 +117,16 @@ public void shootAtPlayer() {
     public void ezShot(){//still needs code to stop from shooting same tile twice. Consider using code in else statement in hardShot()
         int max = 10;
         int min = 1;
-        Random random = new Random();
-        int randRow = random.nextInt(max - min + 1) + min;
-        int randCol = random.nextInt(max - min + 1) + min;
+        Coordinate coor = new Coordinate(0, 0);
```

Comment:
Hey Jackson! Great job getting this started so early! Just a quick note on this line of code here, in class we talked about different code smells and one of the ones we talked about was when you have names that are not descriptive. Since we need to do refactoring on this project I would suggest renaming this to be a little more descriptive. It is an abbreviation of coordinate but later on in the code it we might forget what it means. This is just a thought you do not have to do this.